app/helpers/foreman_monitoring/hosts_helper.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module ForemanMonitoring
+  module HostsHelper
+    def monitoring_hosts_multiple_actions
+      actions = []
+      actions << { action: [_('Set downtime'), select_multiple_downtime_hosts_path], priority: 1000 } \
+        if authorized_for(:controller => :hosts, :action => :select_multiple_downtime)
+      actions << { action: [_('Change Monitoring Proxy'), select_multiple_monitoring_proxy_hosts_path], priority: 1000 } \
+        if authorized_for(:controller => :hosts, :action => :select_multiple_monitoring_proxy)
```

Comment:
Style wise I'd prefer to avoid such long lines
```suggestion
      if authorized_for(:controller => :hosts, :action => :select_multiple_downtime)
        actions << { action: [_('Set downtime'), select_multiple_downtime_hosts_path], priority: 1000 }
      end
      if authorized_for(:controller => :hosts, :action => :select_multiple_monitoring_proxy)
        actions << { action: [_('Change Monitoring Proxy'), select_multiple_monitoring_proxy_hosts_path], priority: 1000 }
      end
        
```

Reply:
Rubocop seems to prefer this kind. I'll commit the original version for now, we can improve the styling in a separate PR.